integration-tests/tests/src/utils/generators.ts

```diff
@@ -15,10 +15,34 @@
 // limitations under the License.
 //
 ////////////////////////////////////////////////////////////////////////////
+
 export function generatePartition() {
   return "xxxxxxxxxxxx4xxxyxxxxxxxxxxxxxxx".replace(/[xy]/g, function (c) {
     const r = (Math.random() * 16) | 0,
       v = c == "x" ? r : (r & 0x3) | 0x8;
     return v.toString(16);
   });
 }
+
+export function uuid() {
```

Comment:
Could this be merged with the `generatePartition` above?

Reply:
Or replaced with `Realm.BSON.uuid()`?